### ISSUE TITLE
Rule Violations only appear as Warnings for Vue files, errors for .js files

### ISSUE THREAD

**Issue by aaron-bidwell** [vuejs/eslint-plugin-vue#995]
**Tell us about your environment**	
* **ESLint version:** 6.6.0	
* **"eslint-plugin-eslint-plugin": 6.0.0,
* **Node version:**	 8.11.1

**Please show your full configuration:**

```json
{
  "env": {
    "node": true,
    "jquery": true
  },

  "globals": {
    "ACC": true,
    "Vue": true
  },

  "rules": {
    "vue/no-async-in-computed-properties": ["error"]
  },
  "extends": [
    "eslint:recommended",
    "plugin:vue/base"
  ],
  "parser": "vue-eslint-parser",
  "parserOptions": {
    "ecmaVersion": 6,
    "sourceType": "module",
    "parser": "babel-eslint"
  }
}

```

**What did you do?**	

```vue

```

**What did you expect to happen?**
I set individual rules or eslint recommend and errors appears for js files but show as only warnings for .vue files.

**Comment by ota-meshi** [comment]
Thank you for this issue!

If you are using the CLI, see the following document:
https://eslint.vuejs.org/user-guide/#running-eslint-from-command-line

If you are using the VSCode, see the following document:
https://eslint.vuejs.org/user-guide/#visual-studio-code

If the problem persists, please tell me how you ran ESLint.

**Comment by ota-meshi** [comment]
I may have misunderstood your issue.

Do you want to switch between errors and warnings for each .vue and .js extension?

If so, I think it should be set using the overrides section.
https://eslint.org/docs/user-guide/configuring#configuration-based-on-glob-patterns

**Comment by aaron-bidwell** [comment]
@ota-meshi thanks for the response and basically when I run eslint through a grunt command it only shows rule violations as warnings for .vue files when I want all .js/.vue violations to be shown as errors

**Comment by ota-meshi** [comment]
I've never used `grunt` so I'm not sure. Does CLI have the same problem?

**Comment by ota-meshi** [comment]
If you don't have the same problem using CLI, you probably have a problem with `grunt` or how to use `grunt`. If you have the same problem with CLI, please let me know.
